protoc-gen-eventuals/protoc-gen-eventuals.py

```diff
@@ -41,31 +41,51 @@ def process_file(self, proto_file: FileDescriptorProto):
 
         file_digest = self.analyze_file(proto_file)
 
-        proto_file_name = file_digest.pop('proto_file_name')
+        proto_file_path = file_digest.pop('proto_file_name')
+
+        index = len(proto_file_path)
+        for ch in reversed(proto_file_path):
+            if (ch == '/'):
+                break
+            index -= 1
+
+        proto_file_name = proto_file_path[index:]
 
         header_file_name = proto_file_name.replace('.proto', '.eventuals.h')
         source_file_name = proto_file_name.replace('.proto', '.eventuals.cc')
 
         eventuals_data = {
             'eventuals_header': header_file_name,
-            'grpc_pb_header': proto_file_name.replace('.proto', '.grpc.pb.h'),
+            'grpc_pb_header': '../' + proto_file_name.replace('.proto', '.grpc.pb.h'),
+            'service_number': 0,
+            'method_number': 0,
```

Comment:
Declare these `None` or omit them here. They are not incremented, but assigned later. 
In jinja, using `None` would also allow for patterns like `service_number | default 0`.

Reply:
Actually, reading further, I would omit these variables here. The rest of the variables in this `dict` is generic, these are changing non-generic ones. There is little won in terms of performance in merging the two, there is some lost clarity and maintainability in merging, so let's keep it seperate.